src/views/Home/index.tsx

```diff
@@ -185,7 +235,11 @@ const HomeView = (): JSX.Element => {
                 Object.entries(filteredPrefabs).map(([key, value]) =>
                   <Home.PrefabsWrapper key={`wrapper-${key}`} isActive={value.active}>
                     <Home.PrefabsPackWrapper isActive={value.active} onClick={() => togglePrefabPack(key)}>
-                      <Home.PrefabsTypeTitle>{`${value.pack.name} (${value.prefabs.length})`}</Home.PrefabsTypeTitle>
+                      <Home.PrefabsTypeTitle style={{
```

Comment:
Plz do not use in-line styles, use props and pass them to styled<T>(x) and then use css method as in other occurences